serve.json

```diff
@@ -1,4 +1,5 @@
 {
+  "cleanUrls": false,
```

Comment:
We renamed folder under `/samples/` with dots in their name. Without `cleanUrls`, `serve-handler` won't load them (because it has weird default page logic.)